_includes/markdown/CSS.md

```diff
@@ -4,377 +4,578 @@ At 10up, we value content and the experience users will have reading it. We writ
 
-```css
-/**
- * Section title
- *
- * Description of section
- */
-```
+Be cautious about which styles you place at this level. Only include styles that should apply to all elements in 99.9% of cases. Removing base-level styles once a project is live—especially if thousands of pages depend on them—can introduce regressions.
 
-For single selectors or inline comments, use this syntax:
+### Components
 
-```css
-/* Inline comment */
-```
+The Component layer is where styles are applied to reusable UI elements. These are more complex than base-level styles and use classes to ensure component-level specificity.
+
+#### Key Guidelines
+
+* Reusability: Focus on creating modular, reusable components. Always consider how the component will respond to:
+
+  * Changes in viewport width.
+
+  * Varying content. For example, if you're styling a card with a title that fits nicely on one line in the design, think about how the card will react to a very long title, or if there’s no title at all.
+
+* Inheritance from Base: Components should take advantage of the solid foundation provided by base styles. When base-level styles are set up correctly, you minimize the need to redefine styles. Usually, you’ll only need to adjust specific properties like colors or spacing at the component level.
+
+* Avoid Using Margins on Components: Components should be built in isolation, meaning we don’t know exactly where a component will be used. Including margins directly in component styles can create unnecessary complications. We'll cover alternative approaches later in this document.
+
+* Encapsulation: Component styles should be scoped and encapsulated to avoid affecting other components. Use methodologies like BEM or CUBE to ensure your styles remain modular and isolated.
+
+* Leverage Variables: Use globally defined variables for properties like colors and spacing. This avoids magic numbers and ensures visual consistency across your app.
+
+* Responsiveness: Components should be fully responsive, adapting effortlessly to different container sizes and screen widths.
 
-Make sure to comment any complex selector or rule you write. For example:
+### Utilities
+
+Utility classes provide quick, reusable styling solutions for common, single-purpose tasks. They allow you to apply styles without the need for creating new components.
+
+#### Key Guidelines
+
+* Single Responsibility: Each utility class should only do one thing. For example, is-style-h1 should only apply styles required to make the text appear as heading level 1 and do nothing else.
+
+* Specificity Level: Utility classes sit at the top of inverted triangle architecture, so they can have higher specificity then base styles and components. In practice they usually have the same level of specificity as components, because of that they should be applied after the component class.
```

Comment:
```suggestion
* Specificity Level: Utility classes sit at the top of the inverted triangle architecture, so they can have higher specificity than base styles and components. In practice, they usually have the same level of specificity as components, because of that they should be applied after the component class.
```